app/Models/BeatmapDiscussion.php

```diff
@@ -124,6 +125,20 @@ public static function search($rawParams = [])
             $params['message_types'] = array_keys(static::MESSAGE_TYPES);
         }
 
+        $params['status'] = static::getValidStatus($rawParams['status'] ?? null);
```

Comment:
looks like this and some other "status"es should be called "beatmapset status"? Otherwise it looks like it's about status of the discussion...